Make Targeting Opt-in (#14462)
pimcore/pimcore: doc/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

      - Print related Events have been moved from into PimcoreWebToPrintBundle. Please check and adapt the Events' namespaces.
    - [Personalization and targeting] has been moved to `pimcore/personalization` package.
      - Config `pimcore:targeting:` has been removed, please use `pimcore_personalization.targeting` in the PimcorePersonalizationBundle instead.

>       - Targeting is now using the opt-in approach and will not be enabled by default. So, user should set the following cookie `pimcore_targeting_enabled=1` to enable targeting. User can use tools like`Cookiebot` to choose consent and set the cookie accordingly.
>        Targeting can also be enabled by setting the following config.
>       ```yaml
>       pimcore_personalization:
>           targeting:
>               enabled: true
>       ```
>       

Reviewer: ```suggestion
      - Targeting is now using the opt-in approach and will not be enabled by default. Add following config to enable it:
      ```yaml
      pimcore_personalization:
          targeting:
              enabled: true
      ```
      Also, a user can use cookie `pimcore_targeting_enabled=1` to enable targeting.
```

        valid_languages:
            - en
            - de

> ```

Reviewer: please revert this change

- Moved implementation of `PimcoreBundleAdminSupportInterface` from `AbstractPimcoreBundle` to bundle classes.
    Moved `getJsPaths`, `getCssPaths`, `getEditmodeJsPaths` and `getEditmodeCssPaths` from `AbstractPimcoreBundle` to `BundleAdminSupportTrait`.
- [Cache] Responses containing a header `Cache-Control: no-cache`, `Cache-Control: private` or `Cache-Control: no-store` will no longer be cached by the full page cache.

> - [Targeting] Targeting is now using the opt-in approach and will not be enabled by default. So, user should set the following cookie `pimcore_targeting_enabled=1` to enable targeting. User can use tools like `Cookiebot` to choose consent and set the cookie accordingly.

Reviewer: Move this under `- [Personalization and targeting]` and add sample config to enable targeting.
